sklearn/metrics/cluster/_unsupervised.py

```diff
@@ -42,6 +44,15 @@ def check_number_of_labels(n_labels, n_samples):
         )
 
 
+@validate_params(
+    {
+        "X": ["array-like", "sparse matrix"],
+        "labels": ["array-like"],
+        "metric": [StrOptions(set(_VALID_METRICS) | {"precomputed"}), callable],
+        "sample_size": [Interval(Integral, 0, 2**32 - 1, closed="both"), None],
```

Comment:
```suggestion
        "sample_size": [Interval(Integral, 1, None, closed="left"), None],
```

Reply:
Thanks for a suggestion, will use it in future PRs.